<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="13075" started-at="2017-11-14T15:46:03Z" finished-at="2017-11-14T15:46:16Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="13075" started-at="2017-11-14T15:46:03Z" finished-at="2017-11-14T15:46:16Z">
      <class name="Ecommerce.FlipkartTest.AppTest1">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:Ecommerce.FlipkartTest.AppTest1@5d5eef3d]" name="beforeTest" is-config="true" duration-ms="13032" started-at="2017-11-14T21:16:03Z" finished-at="2017-11-14T21:16:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="AppTest()[pri:0, instance:Ecommerce.FlipkartTest.AppTest1@5d5eef3d]" name="AppTest" duration-ms="7" started-at="2017-11-14T21:16:16Z" finished-at="2017-11-14T21:16:16Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[Test not implemented]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: Test not implemented
	at Ecommerce.FlipkartTest.AppTest1.AppTest(AppTest1.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AppTest -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:Ecommerce.FlipkartTest.AppTest1@5d5eef3d]" name="afterTest" is-config="true" duration-ms="0" started-at="2017-11-14T21:16:16Z" finished-at="2017-11-14T21:16:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- Ecommerce.FlipkartTest.AppTest1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
